jsdocConfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "tags": {
+    "allowUnknownTags": true
+  },
+  "opts": {
+    "destination": "./docs"
```

Comment:
I don't believe we should commit the JS doc folder to master, but allow Travis to build it via master-dist. The Travis build is already setup to build the jsDoc via PR #12.